### DISCUSSION TITLE
Figma Design System

### DISCUSSION THREAD

**Discussion by franciscoabenza** [material-components/material-web#5065] [Q&A]
Please update the web components on the M3 Design file 🙏

**Comment by asyncliz** [comment]
Can you add more details about what is not accurate with the web components compared to the Figma stickersheet?

**Reply by sachin-dabas** [reply]
I cant find the web components anywhere in Figma. Can you share where to find them? All the components are for Mobile in Material design 3

**Reply by asyncliz** [reply]
https://www.figma.com/community/file/1035203688168086460

**Reply by sachin-dabas** [reply]
Thank you but this doesn't have the web components. It has Mobile components

**Comment by franciscoabenza** [comment]
You are not going to like it, but I can't remember 🤦🏻. I will close and reopen if I encounter the issue again :)
